.github/workflows/test.yml

```diff
@@ -0,0 +1,89 @@
+name: Quality Assurance
+
+on:
+  push:
+  pull_request:
```

Comment:
```suggestion
  pull_request:
  
  # Default: none
  permissions: {}
```

Reply:
Put the `permissions` block without indentation.